src/Models/Payment.php

```diff
@@ -58,6 +70,8 @@ protected static function boot()
                 ->that($payment->order_id)->notEmpty('A payment must be applied to an order.')
```

Comment:
With location id added to the payment record, we could allow order_id to be nullable.  Its no longer a critical source of information to manage / refund the payment.

Reply:
I need to think about this. I need it somewhere (maybe order_items) since locations can be different LLC's and for accounting and taxes, we need to know where the revenue for the order should go.